components/browser/engine-gecko-beta/src/test/java/mozilla/components/browser/engine/gecko/GeckoEngineViewTest.kt

```diff
@@ -275,4 +276,20 @@ class GeckoEngineViewTest {
         engineView.visibility = View.GONE
         verify(engineView.currentGeckoView)?.visibility = View.GONE
     }
+
+    @Test
+    fun `canClearSelection should return false for null selection or null selection text`() {
+        val engineView = GeckoEngineView(context)
+        engineView.currentGeckoView = mock()
+        engineView.currentSelection = mock()
+
+        // null selection returns false
+        whenever(engineView.currentSelection?.selection).thenReturn(null)
+        Assert.assertFalse(engineView.canClearSelection())
```

Comment:
nit: We usually `import org.junit.Assert.assertFalse` so this is just `assertFalse`.